internal/pkg/deploy/cloudformation/stack/validate.go

```diff
@@ -392,3 +395,43 @@ func validateRootDirPath(input string) error {
 func validateContainerPath(input string) error {
 	return validatePath(input, maxDockerContainerPathLength)
 }
+
+func validateName(name *string) error {
```

Comment:
Maybe it would be better if we make the name more specific?
```suggestion
func validatePubSubName(name *string) error {
```

Reply:
Adjusted for clarity.